src/components/forms/formr-part-b/Create.cy-test.tsx

```diff
@@ -82,7 +82,7 @@ describe("Create form B", () => {
       "Section 1: Doctor's details"
     );
     cy.get(
-      ":nth-child(4) > .nhsuk-card__content > :nth-child(2) > :nth-child(1) > .nhsuk-summary-list__value"
+      ":nth-child(3) > .nhsuk-card > .nhsuk-card__content > .nhsuk-summary-list > :nth-child(1) > .nhsuk-summary-list__value"
```

Comment:
Doris identified that this test was actually referencing the View component; so fixed here :)

Reply:
Nice work @DorisWongNHS !